samples/IdentitySample.Mvc/Startup.cs

```diff
@@ -7,6 +7,10 @@
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Logging;
 using Microsoft.Framework.Logging.Console;
+#if DNX451
+using NLog.Config;
```

Comment:
I looked on the NLog github site and there is no issue reported. I need to file a bug but the guidelines suggest I need to provide a repro. I'll get one up using the public CTP6 bits and open the bug.

As a side, they do use Thread Local storage to store the scopes and as you pointed out, that is blowing up for our async calls